docs/commands/ft.create.md

```diff
@@ -1,238 +1,293 @@
+When adding Chinese language documents, set `LANGUAGE chinese` for the indexer to properly tokenize the terms. If you use the default language, then search terms are extracted based on punctuation characters and whitespace. The Chinese language tokenizer makes use of a segmentation algorithm (via [Friso](https://github.com/lionsoul2014/friso)), which segments text and checks it against a predefined dictionary. See [Stemming](/redisearch/reference/stemming) for more information.
+</details>
 
-{{% alert title="Note about deleting a temporary index" color="warning" %}}
-When dropped, a temporary index does not delete the hashes as they may have been indexed in several indexes. Adding the `DD` flag will delete the hashes as well.
-{{% /alert %}}
+<details open>
+<summary><code>SCORE {default_score}</code></summary> 
 
-* **NOHL**: Conserves storage space and memory by disabling highlighting support. If set, we do
-  not store corresponding byte offsets for term positions. `NOHL` is also implied by `NOOFFSETS`.
+is default score for documents in the index. Default score is 1.0.
+</details>
 
-* **NOFIELDS**: If set, we do not store attribute bits for each term. Saves memory, does not allow
-  filtering by specific attributes.
+<details open>
+<summary><code>SCORE_FIELD {score_attribute}</code></summary> 
 
-* **NOFREQS**: If set, we avoid saving the term frequencies in the index. This saves
-  memory but does not allow sorting based on the frequencies of a given term within
-  the document.
+is document attribute that you use as the document rank based on the user ranking. Ranking must be between 0.0 and 1.0. If not set, the default score is 1.
+</details>
 
-* **STOPWORDS {count}**: If set, we set the index with a custom stopword list, to be ignored during
-  indexing and search time. {count} is the number of stopwords, followed by a list of stopword
-  arguments exactly the length of {count}.
+<details open>
+<summary><code>PAYLOAD_FIELD {payload_attribute}</code></summary> 
 
-    If not set, we take the default list of stopwords.
+is document attribute that you use as a binary safe payload string to the document that can be evaluated at query time by a custom scoring function or retrieved to the client.
+</details>
 
-    If **{count}** is set to 0, the index will not have stopwords.
+<details open>
+<summary><code>STOPWORDS {count}</code></summary> 
 
-* **SKIPINITIALSCAN**: If set, we do not scan and index.
+sets the index with a custom stopword list, to be ignored during indexing and search time. `{count}` is the number of stopwords, followed by a list of stopword
+  arguments exactly the length of `{count}`. If `{count}` is set to 0, the index does not have stopwords.
+</details>
 
```

Comment:
Seems redundant (similar content about `STOPWORDS` in line 145 with better explanation)